tests/performance_tests/performance_test_basic.py

```diff
@@ -29,7 +29,7 @@ def waitForEmptyBlocks(node):
     return node.getHeadBlockNum()
 
 appArgs = AppArgs()
-appArgs.add(flag="--save-json", type=bool, default=False, help="Whether to save stats as json")
+appArgs.add(flag = "--save-json", type=str, default="data.json", help="Path to save json output")
```

Comment:
Looks like most places aren't using spaces around the `=` in these cases, thus `flag="--save-json" might be more normalized with the rest of the codebase.  Also, most places put the default case as the last argument.